tests/test_basic.py

```diff
@@ -0,0 +1,223 @@
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "2018-11-14 00:00:00" in ret.stdout,  "time value is printed to console"
+
+def test_gml_time_invalid(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/clc_1000_PT.gml', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_shp_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/Abgrabungen_Kreis_Kleve_Shape.shp', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_dbf_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/Abgrabungen_Kreis_Kleve_Shape.dbf', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_json_bbox(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/schutzhuetten_aachen.json', 
+        '--detail', 'bbox')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout, "bbox is printed to console"
+
+def test_json_time(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/schutzhuetten_aachen.json', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
+    assert "2018-11-14 00:00:00" in ret.stdout,  "time value is printed to console"
+
+def test_json_time_invalid(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata/schutzhuetten_aachen.json', 
+        '--detail', 'bbox',
+        '--time')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == 'invalid time format', "stderr should not be empty"
+
+def test_folder_multiple_files(script_runner):
+    ret = script_runner.run('python3', 'geoextent',
+        '--path', 'Testdata')
+    #print(ret.stdout)
+    assert ret.success, "process should return success"
+    assert ret.stderr == '', "stderr should be empty"
```

Comment:
The process should also return a bounding box containing _all_ datasets!

Reply:
Done.